client/src/EntryView.jsx

```diff
@@ -2,32 +2,71 @@ export default function EntryView() {
   return (
     <>
       <div className="row">
-        <h1>New Entry</h1>
-        <PreviewPhoto />
-        <Form />
-        {/*<SaveButton />*/}
+        <div className="column-full d-flex justify-between">
+          <h1>New Entry</h1>
+        </div>
       </div>
+      <Form />
+      <DeleteAndSaveButton />
     </>
   );
 }
 
+function Form() {
+  return (
+    <form>
+      <div className="row margin-bottom-1">
+        <div className="column-half">
+          <PreviewPhoto />
+        </div>
+        <div className="column-half">
+          <label className="margin-bottom-1 d-block">Title</label>
+          <input
+            type="text"
+            className="input-b-color text-padding input-b-radius purple-outline input-height margin-bottom-2 d-block width-100"
+          />
+          <label className="margin-bottom-1 d-block">Photo URL</label>
+          <input
+            type="text"
+            className="input-b-color text-padding input-b-radius purple-outline input-height margin-bottom-2 d-block width-100"
+          />
+        </div>
+      </div>
+      <div className="row margin-bottom-1">
+        <div className="column-full">
+          <label className="margin-bottom-1 d-block">Notes</label>
+          <textarea
+            className="input-b-color text-padding input-b-radius purple-outline d-block width-100"
+            cols="30"
+            rows="10"></textarea>
+        </div>
+      </div>
+    </form>
+  );
+}
+
 function PreviewPhoto() {
   return (
     <>
-      <img src="./images/placeholder-image-square.jpg" />
+      <img
+        className="input-b-radius form-image"
+        src="./images/placeholder-image-square.jpg"
+      />
     </>
   );
 }
 
-function Form() {
+function DeleteAndSaveButton() {
```

Comment:
Hmm, not sure were you are going with this yet! But I'm assuming conditionally rendering a delete or save button is the better route!

Reply:
🚨 We usually want React to control which components are currently displayed on the  DOM instead of CSS as default! (unless there's a needed use case for CSS)